src/lib/Service/SignalSlotService.php

```diff
@@ -133,7 +133,11 @@ public function hideLocation(int $locationId, bool $isChild = false): void
         }
 
         $content = $this->getContent($location->contentId);
-
+        
+        if (!isset($content)) {
```

Comment:
```suggestion
        if (!$content instanceof Content) {
```

Either that or `if ($content === null) {`